docs/docsite/rst/porting_guides/porting_guide_4.rst

```diff
@@ -0,0 +1,207 @@
+..
+   THIS DOCUMENT IS AUTOMATICALLY GENERATED BY ANTSIBULL! PLEASE DO NOT EDIT MANUALLY! (YOU PROBABLY WANT TO EDIT porting_guide_base_2.11.rst)
+
+.. _porting_4_guide:
+
+=======================
+Ansible 4 Porting Guide
+=======================
+
+.. contents::
+  :local:
+  :depth: 2
+
+
+If you are upgrading from Ansible 2, please first consult the Ansible 3 porting guide before continuing with the Ansible 4 porting guide.
```

Comment:
```suggestion
If you are upgrading from Ansible 2.x, please first consult the Ansible 3 porting guide before continuing with the Ansible 4 porting guide.
```

Reply:
up to you whether you accept that change. Reading 'Ansible 2' just felt weird to me.

Reply:
Why not being explicit here and mention 2.10? If upgrading from 2.9, also the 2.10 porting guide should also be read, and so on for versions before that.